extensions/blocks/podcast-player/components/header.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { memo } from '@wordpress/element';
+
+const Header = memo( ( { track, children } ) => (
+	<div>
+		{ /* TODO: display info based on `track` */ }
+		{ track ? <div>{ track.title }</div> : null }
```

Comment:
Question: should we bail out entirely and just return `null` for this component if the track doesn't exist?

Reply:
I've made this as a really dumb placeholder for now and a proper implementation should rise in the task "Podcast Player Header - connect header/desc markup with API"